bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/DigitalInvoiceScreenTests.kt

```diff
@@ -0,0 +1,199 @@
+        digitalInvoiceScreen.checkForReturnReasonsList()
+    }
+
+    @Test
+    fun test4_verifyCountOnReturnReasonsList() {
+        test3_returnReasonDisplaysWhenToggleSwitchIsDisabled()
+        digitalInvoiceScreen.checkItemCountOnReturnReasonsList()
+    }
+
+    @Test
+    fun test5_clickOnItemFromReturnReasonsList() {
+        test3_returnReasonDisplaysWhenToggleSwitchIsDisabled()
+        digitalInvoiceScreen.clickItemOnReturnReasonsList()
+        idlingResource.waitForIdle()
+        val isItemEnabled = digitalInvoiceScreen.checkItemIsDisabledFromDigitalScreen()
+        assertEquals(false, isItemEnabled)
+    }
+
+    @Test
+    fun test6_enableToggleSwitchToAddItemBackToList() {
+        test5_clickOnItemFromReturnReasonsList()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertDigitalInvoiceText()
+        digitalInvoiceScreen.clickArticleSwitch()
+        val isItemDisabled =digitalInvoiceScreen.checkItemIsEnabledFromDigitalScreen()
+        assertEquals(true, isItemDisabled)
+    }
+
+    @Test
+    fun test7_differenceInTotalAmountWithSwitchEnabledOrDisabled() {
+        test5_clickOnItemFromReturnReasonsList()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.checkTotalTitleIsDisplayed()
+        digitalInvoiceScreen.checkTotalPriceIsDisplayed()
+        digitalInvoiceScreen.storeInitialPrice()
+        digitalInvoiceScreen.clickArticleSwitch()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.checkTotalPriceIsDisplayed()
+        digitalInvoiceScreen.storeUpdatedPrice()
+        digitalInvoiceScreen.assertPriceHasChanged()
+        idlingResource.waitForIdle()
+    }
+
+    @Test
+    fun test8_verifyAdditionalChargesOnDigitalInvoice() {
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertOtherChargesDisplayed()
+    }
+
+    @Test
+    fun test9_clickProceedButtonOnDigitalInvoiceScreen() {
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertDigitalInvoiceText()
+        digitalInvoiceScreen.clickProceedButton()
+        extractionScreen.checkTransferSummaryButtonIsClickable()
```

Comment:
[Clarification] Same question related to the method name... `clickProceedButtonOnDigitalInvoiceScreen` suggests that a click is performed on a `proceedButton`. Ideally we should check either if that button is clickable, or the result of the click.

If we're interested in the result of the click, could we update the test name to something like `checkTransferSummaryButtonIsClickableAfterClickOnProceedButton` ? 

Otherwise, can we check if `proceesButton` is clickable?

Reply:
I can do both of these, but ideally I believe that the changing of name should be opted..

Reason: once the `proceedButton` is clicked, user should navigate to `TransferSummary` screen and `transferSummaryButton` should be enabled on it.